src/Microsoft.Fx.Portability/BreakingChange.cs

```diff
@@ -87,7 +87,7 @@ public int CompareTo(BreakingChange other)
 
         public override int GetHashCode()
         {
-            return Id.GetHashCode();
+            return Id?.GetHashCode() ?? 0;
```

Comment:
The [documentation](https://msdn.microsoft.com/en-us/library/system.object.gethashcode(v=vs.100).aspx#Remarks) is confusing me. :( I didn't think so either but under **Notes to Implementers**:

>  A hash function is used to quickly generate a number (hash code) that corresponds to the value of an object. Hash functions are usually specific to each Type and, for uniqueness, must use at least one of the instance fields as input.
> 
> A hash function must have the following properties:
> 
> * If two objects compare as equal, the GetHashCode method for each object must return the same value. However, if two objects do not compare as equal, the GetHashCode methods for the two object do not have to return different values.